api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -755,4 +761,172 @@ public void copyFile_shouldCopyInputstreamToOutputstreamAndCloseTheOutputstream(
 		verify(output, times(1)).close();
 	}
 
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
+	@Test
+	public void isTestMode_shouldPassIfFunctionalTestModeIsEnabled() {
```

Comment:
rename test method to `isTestMode_shouldReturnTrueIfFunctionalTestModeIsEnabled`

because `shouldPass` is not very specific, every test should pass.